Incorrect import in octavia tests

The import line for the exceptions in the octavia tests are incorrect, missing an 's' on importing the exceptions module. This results in tracebacks such as the following:

```
2022-03-14 18:51:12.266095 | focal-medium | 2022-03-14 18:51:12 [INFO] ----------------------------------------------------------------------
2022-03-14 18:51:12.266399 | focal-medium | 2022-03-14 18:51:12 [INFO] Ran 1 test in 83.179s
2022-03-14 18:51:12.266657 | focal-medium | 2022-03-14 18:51:12 [INFO] OK
2022-03-14 18:51:12.266953 | focal-medium | 2022-03-14 18:51:12 [INFO] ## Running Test zaza.openstack.charm_tests.policyd.tests.KeystoneTests ##
2022-03-14 18:51:12.273631 | focal-medium | 2022-03-14 18:51:12 [ERROR] {'default_alias': 'zaza-229685da8ecf'}
2022-03-14 18:51:12.273780 | focal-medium | 2022-03-14 18:51:12 [ERROR] Model default_alias (zaza-229685da8ecf)
2022-03-14 18:51:13.484063 | focal-medium | Traceback (most recent call last):
2022-03-14 18:51:13.484185 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/bin/functest-run-suite", line 8, in <module>
2022-03-14 18:51:13.484206 | focal-medium |     sys.exit(main())
2022-03-14 18:51:13.484228 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/site-packages/zaza/charm_lifecycle/func_test_runner.py", line 374, in main
2022-03-14 18:51:13.484254 | focal-medium |     func_test_runner(
2022-03-14 18:51:13.484266 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/site-packages/zaza/charm_lifecycle/func_test_runner.py", line 287, in func_test_runner
2022-03-14 18:51:13.484282 | focal-medium |     run_env_deployment(env_deployment, keep_model=preserve_model,
2022-03-14 18:51:13.484289 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/site-packages/zaza/charm_lifecycle/func_test_runner.py", line 161, in run_env_deployment
2022-03-14 18:51:13.484296 | focal-medium |     test.test(
2022-03-14 18:51:13.484310 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/site-packages/zaza/charm_lifecycle/test.py", line 135, in test
2022-03-14 18:51:13.484317 | focal-medium |     run_test_list(tests)
2022-03-14 18:51:13.484324 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/site-packages/zaza/charm_lifecycle/test.py", line 118, in run_test_list
2022-03-14 18:51:13.484331 | focal-medium |     testcase = utils.get_class(_testcase)
2022-03-14 18:51:13.484348 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/site-packages/zaza/charm_lifecycle/utils.py", line 483, in get_class
2022-03-14 18:51:13.484356 | focal-medium |     module = importlib.import_module(module_name)
2022-03-14 18:51:13.484363 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/importlib/__init__.py", line 127, in import_module
2022-03-14 18:51:13.484370 | focal-medium |     return _bootstrap._gcd_import(name[level:], package, level)
2022-03-14 18:51:13.484377 | focal-medium |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
2022-03-14 18:51:13.484385 | focal-medium |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
2022-03-14 18:51:13.484393 | focal-medium |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
2022-03-14 18:51:13.484400 | focal-medium |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
2022-03-14 18:51:13.484406 | focal-medium |   File "<frozen importlib._bootstrap_external>", line 848, in exec_module
2022-03-14 18:51:13.484416 | focal-medium |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
2022-03-14 18:51:13.484424 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/site-packages/zaza/openstack/charm_tests/policyd/tests.py", line 54, in <module>
2022-03-14 18:51:13.484431 | focal-medium |     import zaza.openstack.charm_tests.octavia.tests as octavia_tests
2022-03-14 18:51:13.484438 | focal-medium |   File "/home/ubuntu/src/review.opendev.org/openstack/charm-keystone/.tox/func-target/lib/python3.8/site-packages/zaza/openstack/charm_tests/octavia/tests.py", line 32, in <module>
2022-03-14 18:51:13.484460 | focal-medium |     from zaza.openstack.utilities.exception import (
2022-03-14 18:51:13.484467 | focal-medium | ModuleNotFoundError: No module named 'zaza.openstack.utilities.exception'
```

--- a/zaza/openstack/charm_tests/octavia/tests.py
+++ b/zaza/openstack/charm_tests/octavia/tests.py
@@ -29,7 +29,7 @@ import zaza.openstack.utilities.openstack as openstack_utils
 
 from zaza.openstack.utilities import generic as generic_utils
 from zaza.openstack.utilities import ObjectRetrierWraps
-from zaza.openstack.utilities.exception import (
+from zaza.openstack.utilities.exceptions import (
     LoadBalancerUnexpectedState,
     LoadBalancerUnrecoverableError,
 )

